lambda/file_format.tf

```diff
@@ -11,10 +11,15 @@ resource "aws_lambda_function" "file_format_lambda_function" {
   tags                           = var.common_tags
   environment {
     variables = {
-      ENVIRONMENT    = aws_kms_ciphertext.environment_vars_file_format["environment"].ciphertext_blob
-      INPUT_QUEUE    = aws_kms_ciphertext.environment_vars_file_format["input_queue"].ciphertext_blob
-      OUTPUT_QUEUE   = aws_kms_ciphertext.environment_vars_file_format["output_queue"].ciphertext_blob
-      ROOT_DIRECTORY = aws_kms_ciphertext.environment_vars_file_format["root_directory"].ciphertext_blob
+      ENVIRONMENT                 = aws_kms_ciphertext.environment_vars_file_format["environment"].ciphertext_blob
+      INPUT_QUEUE                 = aws_kms_ciphertext.environment_vars_file_format["input_queue"].ciphertext_blob
+      OUTPUT_QUEUE                = aws_kms_ciphertext.environment_vars_file_format["output_queue"].ciphertext_blob
+      ROOT_DIRECTORY              = aws_kms_ciphertext.environment_vars_file_format["root_directory"].ciphertext_blob
+      S3_BUCKET                   = var.upload_bucket
+      OUTPUT_SQS_QUEUE            = local.api_update_queue_url
+      INPUT_SQS_QUEUE             = local.file_format_queue_url
+      CONTAINER_SIGNATURE_VERSION = "20220704"
+      DROID_SIGNATURE_VERSION     = "V107"
```

Comment:
I just want to understand the environment variables CONTAINER_SIGNATURE_VERSION and DROID_SIGNATURE_VERSION. Do we really need to hardcode it here? Can we create variables for this?

Reply:
These versions are set by the Pronom team every time they release a new version of the signature files so it's not something we control which is why they're hard-coded. The way the new lambda is going to work is it will download the signature files it needs when it runs. They're too large to include in the lambda code.

So the idea behind this is that when we need to upgrade the signature files, we only need to change it in terraform and then each new lambda will pick up the new versions. 

I think this makes sense but let me know if it doesn't because I've been thinking about it for too long.

Reply:
Yeah, I think it looks fine.